injection/sharedmain/main.go

```diff
@@ -111,6 +114,32 @@ var (
 	WebhookMainWithConfig  = MainWithConfig
 )
 
+// MainNamed runs the generic main flow for controllers and webhooks.
```

Comment:
should probably say something about the difference between this method and the others here